build.sbt

```diff
@@ -95,6 +95,70 @@ lazy val core = (project in file("core"))
     (compile in Compile) := ((compile in Compile) dependsOn createTargetClassesDir).value
   )
 
+lazy val deltacontribs = (project in file("delta-contribs"))
+  .enablePlugins(GenJavadocPlugin, JavaUnidocPlugin, ScalaUnidocPlugin)
+  .dependsOn(core)
+  .settings (
+    name := "delta-contribs",
+    commonSettings,
+    scalaStyleSettings,
+    unidocSettings,
+    releaseSettings,
+    libraryDependencies ++= Seq(
```

Comment:
do we need to add these dependencies? I think this project automatically gets the dependencies from the core module

Reply:
I have removed almost all of the dependencies but `spark-sql` which is needed.
Because the `spark-sql` dependency is `provided` in the `core` module it is not resolved as a transitive dependency for the `delta-contrib` module.
(If we'll make the `spark-sql` module a `compile` dependency in the core then it works fine but I didn't want to change the `core` module dependencies)

Reply:
adding `provided->provided` solved it.